### PR TITLE
community: fix yuan2 errors in LLMs

### PR DESCRIPTION
**Pull request by cauwulixuan**
1. fix yuan2 errors while invoke Yuan2.
2. update tests.

### CODE REVIEW

File: libs/community/langchain_community/llms/yuan2.py [hunk 1]
```diff
@@ -135,6 +136,13 @@ def _call(
             input = prompt
 
         headers = {"Content-Type": "application/json"}
+        if (self.top_p or 0) > 0 and (self.top_k or 0) > 0:
+            logger.warning(
+                "top_p and top_k cannot be set simultaneously. "
+                "set top_k to 0 instead..."
+            )
+            self.top_k = 0
```
Comment (hwchase17) [inline libs/community/langchain_community/llms/yuan2.py:144]: this should be checked during initialization

Comment (cauwulixuan) [reply libs/community/langchain_community/llms/yuan2.py]: Sure, thanks for your advice. PTAL;)

**Comment by cauwulixuan** [comment]
It seems like the CI progress is failed with some network error.. Could you trigger it again? @hwchase17

**Comment by cauwulixuan** [comment]
Well, all checks have passed. @hwchase17 PTAL, thanks.

**Review by baskaryan — APPROVED** [review]

**Pull request merged**